packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js

```diff
@@ -134,55 +153,75 @@ const CustomSection = section => [
             __html: markdownToHTML(section.readme, `${section.id}-`),
           }}
           bareHtml
+          alternate
         />,
       ])
     : null,
 ];
 
-const DocsPageBuilder = props => (
-  <BpkContentContainer>
-    <Helmet title={props.title} />
-    <Heading level="h1">{props.title}</Heading>
-    {flatten(toNodes(props.blurb))}
-    {props.showMenu && (
-      <BpkList>
-        {flatten(
-          [...props.components, ...props.customSections].map(
-            ExampleNavListItem,
-          ),
-        )}
-      </BpkList>
-    )}
-    {props.tokenMap ? <TokenSwitcher tokens={props.tokenMap} /> : null}
-    {props.usageTable
-      ? flatten([
+const NeoDocsPageBuilder = props => {
```

Comment:
This shouldn't be called `NeoDocsPageBuilder` anymore should it?

Reply:
Ah I missed that, thanks!